libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -221,6 +223,47 @@ AP_Arming::AP_Arming()
     AP_Param::setup_object_defaults(this, var_info);
 }
 
+__INITFUNC__ void AP_Arming::init(void)
+{
+    // PARAM_CONVERSION - 4.7 CHECK -> SKIPCHK
+
+    if (!checks_to_skip.configured()) {
+        // new parameter is not configured (though it may be set non-zero in a
+        // defaults table e.g. on Sub)
+        int32_t skipchk_new = checks_to_skip.get(); // get param default
+
+        int32_t checks_old;
+        bool found = false;
+
+#if APM_BUILD_TYPE(APM_BUILD_ArduPlane)
+        // the old index is inexplicably shifted only on plane, and
+        // get_param_by_index's arg is not wide enough, so we reimplement
+        uint16_t key;
+        if (AP_Param::find_top_level_key_by_pointer(this, key)) {
+            const AP_Param::ConversionInfo type_info = {key, 8 << 6 /* shifted??? */, AP_PARAM_INT32, nullptr };
```

Comment:
this is due to the AP_NESTEDGROUPINFO() in AP_Arming_Plane.cpp, this adds another level of table